packages/vx-react-spring/src/axis/AnimatedTicks.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { animated, useTransition, interpolate } from 'react-spring';
+import cx from 'classnames';
+import Orientation from '@vx/axis/lib/constants/orientation';
+import { TicksRendererProps, AxisScale } from '@vx/axis/lib/types';
+import { Text } from '@vx/text';
+
+import useLineTransitionConfig, {
+  TransitionConfig,
+} from '../spring-configs/useLineTransitionConfig';
+
+export default function AnimatedTicks<Scale extends AxisScale>({
+  hideTicks,
+  horizontal,
+  orientation,
+  scale,
+  tickClassName,
+  tickLabelProps: allTickLabelProps,
+  tickStroke = '#222',
+  tickTransform,
+  ticks,
+  animationTrajectory,
+}: TicksRendererProps<Scale> & Pick<TransitionConfig<Scale>, 'animationTrajectory'>) {
+  const animatedTicks = useTransition(ticks, tick => `${tick.value}-${horizontal}`, {
+    unique: true,
+    ...useLineTransitionConfig({ scale, animateXOrY: horizontal ? 'x' : 'y', animationTrajectory }),
```

Comment:
this + the `<>` wrapper below are the only updates in this file. `AnimatedTicks` was previously a directory and the indentation with `<>` made git think it's a new file